keyboards/edinburgh41/info.json

```diff
@@ -0,0 +1,17 @@
+{

```

Comment:
This is invalid and needs to be fixed.   

Eg, it needs to match the keyboard's layout.